src/client/pmix_client.c

```diff
@@ -552,21 +552,10 @@ PMIX_EXPORT pmix_status_t PMIx_Abort(int flag, const char msg[],
         /* shouldn't be possible */
         goto done;
     }
-#if defined(PMIX_ENABLE_DSTORE) && (PMIX_ENABLE_DSTORE == 1)
```

Comment:
PMI1/PMI2 tests are passing now.
@karasevb, please add them to Mellanox jenkins script.